### PR TITLE
Extended agent functionality

### PR DESCRIPTION
**Pull request by daanwerf**
A very simple start of what we want our agent to accomplish, mostly to show how demolishing is done and how this can be tested. The way this is implemented right now is not the most clean one, but it gets the job done. User story:

As an agent
When in a simulation
I am able to destroy stuff if I think 
this is benificial for me.
To make room for construction or to 
optimize a certain indicator.

In the coming days I will try to add more functionality to this category and try to give the agent a sense of strategy in the things he wants to do.

### CODE REVIEW

File: TU-Agent/tygronInit.mod2g [hunk 1]
```diff
@@ -1,7 +1,7 @@
 use agentKnowledge as beliefs.
 use agentGoals as goals.
 
-module tygronInit {
+module ertygronInit {
```
Comment (tvleest) [inline TU-Agent/tygronInit.mod2g]: is this correct?

**Comment by tvleest** [comment]
I think you have the wrong environment, there's 2 commits missing from the master, might explain the travis fails.

**Comment by MichielvdBerg** [comment]
Now the travis build is green, I approve.

**Comment by jzoon** [comment]
Seems to be working so approved

**Pull request merged**